crates/biome_cli/src/commands/lint.rs

```diff
@@ -65,6 +70,13 @@ pub(crate) fn lint(
     fs_configuration.merge_with(files_configuration);
     fs_configuration.merge_with(vcs_configuration);
 
+    session
+        .app
+        .workspace
+        .update_settings(UpdateSettingsParams {
+            configuration: fs_configuration.clone(),
+        })?;
```

Comment:
I had to to this so when I use the `workspace.is_path_ignored` method later in the `store_changed_files` function, the workspace already has the correct configuration.

Not sure how badly updating the settings twice and doing `fs_configuration.clone()` would affect perf, so please advise if this is okay or not.